daemon/k8s_watcher.go

```diff
@@ -911,16 +950,16 @@ func (d *Daemon) addK8sEndpointV1(ep *v1.Endpoints) {
 			// succeeded.
 			if endpointsEqual {
 				scopedLog.Debugf("no changes to Kubernetes endpoint; not triggering policy updates")
-				return
```

Comment:
This changes the flow of control. The point of this `if` is to not trigger a policy update below.
```
return nil
```